### PR TITLE
Schema Support PostgreSQL Index Column Ordering

### PR DESCRIPTION
**Pull request by s0kil**
Fixes https://github.com/digitallyinduced/ihp/issues/1384

### CODE REVIEW

File: IHP/IDE/CodeGen/MigrationGenerator.hs [hunk 1]
```diff
@@ -214,7 +214,7 @@ migrateTable StatementCreateTable { unsafeGetCreateTable = targetTable } Stateme
                         (Just dropColumn) = actualColumns
                                 |> find \case
                                     Column { name } -> name == columnName
-                                    otherwise       -> False                                
+                                    otherwise       -> False
```
Comment (mpscholten) [inline IHP/IDE/CodeGen/MigrationGenerator.hs:217]: Why was this changed?

Comment (s0kil) [reply IHP/IDE/CodeGen/MigrationGenerator.hs:217]: I'm not sure, it seems the editor reformatted it.

**Review by mpscholten — APPROVED** [review]
👍  thnaks
